docs/upgrade_skaffold.md

```diff
@@ -64,25 +50,17 @@ deploy:
       - ./deployments/snack.yaml
 ```
 
-In Tilt we similarly associate your image and your Kubernetes YAML through the concept of a service:
+In Tilt we associate your image and your Kubernetes YAML by image tag.
 
 ```python
-  yaml = read_file('./deployments/snack.yaml')
-  service = k8s_service(img, yaml=yaml)
+k8s_resource('snack', 'deployments/snack.yaml')
```

Comment:
I think this has made enough room that it'd be more instructive to include:
*) multiple dockerfiles
*) multiple yamls
*) all passed to k8s_resource("deploy', ['snack.yaml', 'vigoda.yaml'])

WDYT?